modules/ecs/cluster/main.tf

```diff
@@ -30,44 +52,115 @@ resource "aws_security_group" "lb_security_group" {
   description = "${aws_ecs_cluster.ecs_cluster.name} load balancers"
   vpc_id      = var.aws_vpc_id
 
+  ingress {
+    protocol  = "tcp"
+    from_port = 80
+    to_port   = 80
+
+    cidr_blocks = [element(var.whitelist, count.index)]
+  }
+
+  ingress {
+    protocol  = "tcp"
+    from_port = 443
+    to_port   = 443
+
+    cidr_blocks = [element(var.whitelist, count.index)]
+  }
+
+  egress {
+    from_port = 0
+    to_port   = 0
+    protocol  = "-1"
+
+    cidr_blocks = [
+      "0.0.0.0/0",
+    ]
+  }
+
   tags = var.tags
 
   lifecycle {
     create_before_destroy = true
   }
 }
 
-resource "aws_security_group_rule" "lb_security_group_all_egress" {
-  type              = "egress"
-  from_port         = 0
-  to_port           = 0
-  protocol          = "-1"
-  cidr_blocks       = ["0.0.0.0/0"]
-  security_group_id = aws_security_group.lb_security_group.id
-}
+data "aws_iam_policy_document" "ecs_runtime" {
```

Comment:
have we tried to apply this anywhere to look at the resulting policy in the web console?

Reply:
Yes I've tested the opposite in parking, they are the same thing basically. The diff didn't change.